src/System.Windows.Forms/src/System/Windows/Forms/ToolStripHighContrastRenderer.cs

```diff
@@ -502,5 +463,48 @@ private static void DrawHightContrastDashedBorder(Graphics graphics, ToolStripIt
             graphics.DrawRectangle(focusPen1, bounds);
             graphics.DrawRectangle(focusPen2, bounds);
         }
+
+        private void RenderItemImageOfLowColorDepth(ToolStripItemImageRenderEventArgs e)
+        {
+            Image? image = e.Image;
+
+            if (image is not null)
+            {
+                Graphics g = e.Graphics;
+
+                ToolStripItem item = e.Item;
+                Rectangle imageRect = e.ImageRectangle;
+                using (ImageAttributes attrs = new ImageAttributes())
+                {
```

Comment:
[nit] Curlies don't look needed here
```suggestion
                using ImageAttributes attrs = new();
```